certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+                  "title": "Start Date",
+                  "x-ms-visibility": "advanced"
+                },
+                "endDate": {
+                  "type": "string",
+                  "description": "endDate",
+                  "title": "End Date",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": [
+                "reciprocalRelationshipName",
+                "relatedToCustomerRecordNumber",
+                "relationshipName"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Organizations/{id}": {
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Update Organization",
+        "description": "Update Organization",
+        "operationId": "Update-Organization",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Organization ID",
+            "x-ms-summary": "Organization ID"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "name"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description"
+                },
+                "showInDirectory": {
+                  "type": "boolean",
+                  "description": "showInDirectory"
+                },
+                "branchName": {
+                  "type": "string",
+                  "description": "branchName"
+                },
+                "website": {
+                  "type": "string",
+                  "description": "website"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Setup/customfields/{pageNumber}": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.